default.nix

```diff
@@ -1,4 +1,6 @@
-{ pkgs ? import <nixpkgs> {}
+{ pkgs ? import nixpkgs {}
+# Use a pinned nixpkgs rather than the one on NIX_PATH
+, nixpkgs ? ./nixpkgs
```

Comment:
Why not do this the same as the other pins below?

Reply:
The other pins (hackage and stackage) can be overridden with a json file because that will be a fairly common thing to do for users -- running `nix-prefetch-git` to select a more recent revision of hackage or stackage from github.

For the pkgs/nixpkgs args I want to use the "normal" pattern for pinning.